### PR TITLE
chore: remove bok-choy usage

### PR DESCRIPTION
**Pull request by salman2013**
Description

As the bok-choy has been deprecated https://github.com/openedx/public-engineering/issues/13, we are removing its usage from the code in this PR.

Ticket : https://github.com/orgs/openedx/projects/55/views/1?pane=issue&itemId=40721593

In this PR i also updated xblockutils' new hierarchy for classes but with support for backward compatibility for Quince earlier releases.

Ticket: https://github.com/orgs/openedx/projects/55/views/1?pane=issue&itemId=41364624

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,11 +23,11 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-image-explorer',
-    version='2.0',
+    version='2.1',
```
Comment (farhan) [inline setup.py:26]: I think it good time to add path version as well
`2.1.0`

File: setup.py [hunk 2]
```diff
@@ -23,11 +23,11 @@ def package_data(pkg, root_list):
...
     description='XBlock - Image Explorer',
     packages=['image_explorer'],
     install_requires=[
-        'XBlock>=1.2',
+        'XBlock>=1.8.1',
```
Comment (farhan) [inline setup.py:30]: Its not required as we have provided backward compatibility

**Review by farhan — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (salman2013) [reply setup.py]: done

File: setup.py (hunk 1 above)
Comment (salman2013) [reply setup.py]: done

**Review by farhan — APPROVED** [review]

**Review by feanil — APPROVED** [review]

**Pull request merged**